### ISSUE TITLE
OpenBSD file: unknown option -- mime-encoding

### ISSUE THREAD

**Issue by meutel** [ultrabug/py3status#1171]
On OpenBSD (6.2-current), py3status fails with an error:
```
file: unknown option -- mime-encoding
usage: file [-bchiLsW] file ...
```
The `file` command on OpenBSD does not have the `--mime-encoding` option. The command is used in: `parse_config.py` ine 536
```
encoding = check_output(['file', '-b', '--mime-encoding', '--dereference', config_path])
encoding = encoding.strip().decode('utf-8')
```
As a workaround, I replaced those lines with hardcoded value:
```
encoding = 'utf-8'
```
then the program runs OK.

Maybe encoding guess should use a python library (chardet?) instead of using the file command. I will try to propose a patch (never coded anything in python, but this is a nice way to learn).

**Comment by tobes** [comment]
Thanks for reporting this and your analysis of the issue

> chardet

We try to keep py3status as light as possible as far as needed packages are concerned.  I've created a fix in PR #1173 that should fix the problem for you.  If you can confirm the fix that would be great

**Comment by meutel** [comment]
Yes, it's fixed. Thanks

**Comment by tobes** [comment]
fixed via  #1173
